src/components/blocks/api-reference/external-references/ApiReferenceBlockquote.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentProps, ReactNode } from 'react';
+import { HtmlComponentProps } from '../../../html-component-props';
+import { Blockquote } from '../../external-references';
+import Html from '../../Html';
+
+const StyledApiReferenceBlockquote = ({
+  attribs,
```

Comment:
I'm a bit concerned with naming `props` `attribs`. Reason being, it's a very non-standard and also HTML already has attributes, which work very differently to props (and in fact, there are differences between using the in HTML and JavaScript). 

I'm worried this would create confusion both for people working between projects and more junior developers.

Why not do `props` or `...props` ?

Reply:
These are just the attributes taken from the HTML as rendered via. textile, so they are intended to work the same way as attributes; for the basic styled blockquote component they happen to be the only props provided, but there could be others.